test/e2e/wdio.conf.js

```diff
@@ -0,0 +1,310 @@
+  // baseUrl: 'www.staging.cloudinary.com',
+  //
+  // Default timeout for all waitFor* commands.
+  waitforTimeout: 60000,
+  //
+  // Default timeout in milliseconds for request
+  // if Selenium Grid doesn't send response
+  connectionRetryTimeout: 60000,
+  //
+// Default request retries count
+  connectionRetryCount: 3,
+  //
+  // Initialize the browser instance with a WebdriverIO plugin. The object should have the
+  // plugin name as key and the desired plugin options as properties. Make sure you have
+  // the plugin installed before running any tests. The following plugins are currently
+  // available:
+  // WebdriverCSS: https://github.com/webdriverio/webdrivercss
+  // WebdriverRTC: https://github.com/webdriverio/webdriverrtc
+  // Browserevent: https://github.com/webdriverio/browserevent
+  // plugins: {
+  //     webdrivercss: {
+  //         screenshotRoot: 'my-shots',
+  //         failedComparisonsRoot: 'diffs',
+  //         misMatchTolerance: 0.05,
+  //         screenWidth: [320,480,640,1024]
+  //     },
+  //     webdriverrtc: {},
+  //     browserevent: {}
+  // },
+  //
+  // Test runner services
+  // Services take over a specific job you don't want to take care of. They enhance
+  // your test setup with almost no effort. Unlike plugins, they don't add new
+  // commands. Instead, they hook themselves up into the test process.
+  // services: [],//
+  // Framework you want to run your specs with.
+  // The following are supported: Mocha, Jasmine, and Cucumber
+  // see also: http://webdriver.io/guide/testrunner/frameworks.html
+  //
+  // Make sure you have the wdio adapter package for the specific framework installed
+  // before running any tests.
+  framework: "mocha",
+  //
+  // Test reporter for stdout.
+  // The only one supported by default is 'dot'
+  // see also: http://webdriver.io/guide/reporters/dot.html
+  // reporters: ['dot', 'allure'],
+  // reporterOptions: {
+  //     allure: {
+  //         outputDir: 'allure-results',
+  //         disableWebdriverStepsReporting: true,
+  //     },
+  // },
+
+  //
+  // Options to be passed to Mocha.
+  // See the full list at http://mochajs.org/
+  mochaOpts: {
+    ui: "bdd",
+    timeout: 300000 // test timeout, test will fall after 5 minutes on timeout
+  },
+  //
+  // =====
+  // Hooks
+  // =====
+  // WebdriverIO provides several hooks you can use to interfere with the test process in order to enhance
+  // it and to build services around it. You can either apply a single function or an array of
+  // methods to it. If one of them returns with a promise, WebdriverIO will wait until that promise got
+  // resolved to continue.
+  /**
+   * Gets executed once before all workers get launched.
+   * @param {Object} config wdio configuration object
+   * @param {Array.<Object>} capabilities list of capabilities details
+   */
+  onPrepare: function(config, capabilities) {
+    /**
+     * Load configurations from .env
+     */
+    dotenv.config();
```

Comment:
Which configurations do you expect to read from `.env`? Anything worth mentioning in README?

Reply:
removed. No use of .env in this project